app/Http/Controllers/EmployeeController.php

```diff
@@ -39,27 +41,29 @@ public function draw(Request $request){
             }
         }
 
+         $employees = Employee::select('employees.*' , 'companies.name as company_name' ,'companies.branch', 'companies.deleted_at as company_deleted_at' , 'bank_accounts.account_no')
+                ->join('companies', 'employees.company_id', '=', 'companies.id')
+                ->join('bank_accounts', 'employees.id', '=', 'bank_accounts.employee_id');
+
+        //dropdown filter not null
         if(!empty($arr)){
             $searchPosition = empty($arr) ? $search : (array_key_exists(4, $arr) ? $arr[4] : null);
```

Comment:
array_key 4 is for what?. can't we get that from client side as a parameter.

Reply:
this issue was fixed on another branch as both issue are same -> PR #83